net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@(PACKAGE_wireless-tools||PACKAGE_iw) +ethtool +procps +CONFIG_USB_SUPPORT:usbutils +CONFIG_PCI_SUPPORT:pciutils
```

Comment:
Now it looks like at least `aircrack-ng` utility from 1.2-rc3 is broken (race condition among threads can lead to fault cracking result).  So I will close this PR for now until at least the release version is out.